src/common/time/instant.ts

```diff
@@ -0,0 +1,143 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {TimeUnit} from './timeunit.js';
+
+/**
+ * An Instant represents a specific instance of universal time with a
+ * defined precision.
+ *
+ * TODO implement comparable?
+ * TODO implement other utility methods
```

Comment:
Time arithmetic would be valuable -- let's consider later.

Reply:
ack.